libcxx/include/__config

```diff
@@ -46,6 +46,12 @@
 #    endif
 #  endif
 
+#  define _LIBCPP_PUSH_MACROS _Pragma("push_macro(\"min\")") _Pragma("push_macro(\"max\")") _Pragma("push_macro(\"refresh()\")") _Pragma("push_macro(\"move(int, int)\")") _Pragma("push_macro(\"erase()\")") _Pragma("push_macro(\"__noinline__\")")
```

Comment:
Is there an open bug we can reference here? Just to mark the noinline code with? (hopefully to encourage us to remove it).